index.html

```diff
@@ -217,5 +217,32 @@ <h4>Skills</h4>
     <hr class="skills">
 </section>
 </main>
+
+<!-- Footer Section -->
+
+<footer class="contact-forms">
+    <div class="contact">
+        <h1 class="form-heading">Contact me</h1> 
+    </div>
+    <div class="form-paragraph">
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    </div>
+    <form action="https://formspree.io/f/mpzbjyrv" method="post">
+        <div class="name">
+            <input name="usr_name" class="name-text" type="text" placeholder="Yeremias" maxlength="30" required >
+        </div>
+        <div class="email">
+            <input name="usr_email" class="email-field" type="email" placeholder="natajayanj14@gmail.com" required>
+        </div>
+        <div class="text-field">
+            <textarea class="text-fld" name="text-field" id="text-field" cols="30" rows="10" placeholder="Write your message here" maxlength="500"></textarea>
+        </div>
+        <div class="form-btn-1">
+            <button type="button" class="form-btn">Get It Touch</button>
```

Comment:
- [ ] Kindly put the correct `type` for the submission button because now the form doesn't submit any data.
[Hint] `type= "submit"`